test/jdl/jhipster/default-application-options.spec.js

```diff
@@ -60,6 +60,9 @@ describe('DefaultApplicationOptions', () => {
             it('should set the client theme variant option to none', () => {
                 expect(options.clientThemeVariant).to.equal('');
             });
+            it('should set withAdminUI option to true', () => {
+                expect(options.withAdminUi).to.equal(true);
```

Comment:
A small improvement: `expect(options.withAdminUi).to.equal(true);` => `expect(options.withAdminUi).to.be.true;`